docs/References.bib

```diff
@@ -97,9 +97,9 @@ @article{Fishbone1976apj
 
 @article{Foucart2015vpa,
       author         = "Foucart, Francois and O'Connor, Evan and Roberts, Luke
-                        and Duez, Matthew D. and Haas, Roland and Kidder, Lawrence
-                        E. and Ott, Christian D. and Pfeiffer, Harald P. and
-                        Scheel, Mark A. and Szilagyi, Bela",
+                        and Duez, Matthew D. and Haas, Roland and Kidder,
+                        Lawrence E. and Ott, Christian D. and Pfeiffer,
+                        Harald P. and Scheel, Mark A. and Szilagyi, Bela",
```

Comment:
Since you're touching this, please fix the indentation of the entry to 2 spaces like the rest of the file. (we really have to add automated checks for these things to keep this file tidy and consistent...)

Reply:
Until there's a check for that I'm not changing it.